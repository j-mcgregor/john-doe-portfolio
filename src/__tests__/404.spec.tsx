import React from 'react'
import renderer from 'react-test-renderer'
import NotFound from '../pages/404'

describe('404', () => {
    it('should render', () => {
        // Assemble
        const banner = renderer.create(<NotFound />)
        const bannerInstance = banner.toJSON()

        // Assert
        expect(bannerInstance).not.toBeNull()
        if (bannerInstance) {
            expect(bannerInstance).toMatchSnapshot()
        }
    })
})
