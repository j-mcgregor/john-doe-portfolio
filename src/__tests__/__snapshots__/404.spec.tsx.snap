// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 should render 1`] = `
Array [
  <nav
    className="navbar"
  >
    <a
      className="brand"
      href="/"
    >
      John Doe
    </a>
    <ul
      className="nav-list"
    >
      <li
        className="nav-item"
      >
        <a
          href="/contact"
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>,
  <div
    className="container-fluid"
  >
    <div
      className="flex flex-center flex-column"
      style={
        Object {
          "height": "calc(90vh - 80px)",
        }
      }
    >
      <h1>
        NOT FOUND
      </h1>
      <p>
        You just hit a route that doesn't exist... the sadness.
      </p>
    </div>
  </div>,
  <footer
    className="footer"
  >
    Reach me on
    <div
      className="social"
    />
  </footer>,
]
`;
